guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
<p>This is the most significant step for fitting a timeseries model. Here, along with the data, the backbone for training the model and the sequence length are passed as the parameters. Out of these, the sequence length has to be selected carefully, as it is important for fitting a timeseries model. The sequence length here would typically be the seasonality of the data. The Timeseries model is initialized as follows:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>